e2e/_suites/fleet/installers.go

```diff
@@ -150,11 +151,14 @@ func (i *DockerPackage) Preinstall() error {
 		return err
 	}
 
+	// wait for tagging to ensure the loaded image is present
+	e2e.Sleep(3 * time.Second)
```

Comment:
I've seen errors on CI with not found images, as load and import executed in the same second.